1904-12-29.xml

```diff
@@ -0,0 +1,3180 @@
+                            delivery in Alesxandria or postage to subscriber's addronn) P.T. 251 per
+                            annum, P.T. 116 for six months, P.T. 80 for three months. To other
+                            countries in the Portal Union P.T. 278 (2.16z.) per annum.</p>
+                    </div>
+                </div>
+                <div type="item">
+                    <p>ADVERTISEMENTS. - P.T. 4 per line. Minimum charge P.T. 20. Births, Marriages,
+                        or Deaths, not exceeding three lines, P.T. 20, every additional line P.T.
+                        10. Notices in news columns P.T. 20 per line.
+                        <?oxy_comment_start author="ajk22" timestamp="20220916T155733-0400" comment="illegible"?>Ourusia<?oxy_comment_end?>
+                        ensured info for standing advertisements.</p>
+                    <p>N.B. -Subscriptions commence from the 1st or 16th of any month. </p>
+                </div>
+                <div type="item">
+                    <p>SUBSCRIPTIONS and ADVEBTISEMENTS are due in advance. P. O. Orders And Cheques
+                        to be made payable to the Editor and Manager, Rowland Snelling, Alexandria </p>
+                    <p>London Office: 86, New Broad. street, E.C.</p>
+                </div>
+                <div type="section">
+                    <p>THE "EGYPTIAN GAZETTE" IS PRINTED ON PAPER</p>
+                    <p>MANUFACTURED AND SUPPLIED BY THE </p>
+                    <p>LONGON PAPER MILLS Co., LIMITED. SALES OFFICE</p>
+                    <p>27, CANNON STREET, E.C. 25-6</p>
+                </div>
+                <div type="section">
+                    <head>The Egyptian Gazette</head>
+                    <p>An English Daily Newspaper, Established in 1880.</p>
+                    <p>Editor &amp; Manager: B. Snelling.</p>
+                    <p> Price One Plastre Tariff.</p>
+                    <p>THURSDAY, DECEMBER 29, 1904.</p>
+                </div>
+                <div type="section">
+                    <head> LAND VALUES IN EGYPT.</head>
+                    <p>We call attention to an article on the above subject by Mr. Lang Anderson in
+                        the last Journal of the British Chamber of Commerce, and which we reproduce
+                        in today's issue. Mr. Anderson is optimistic, but when we follow out his
+                        closely-reasoned and lucid summary of the causes why land should have
+                        increased in value, his conclusion seems perfectly logical—that prices are
+                        not inflated, that the rise has been based on very sound conditions, and
+                        that before long land in a this country will be sold on 20 years' purchase
+                        of the annual revenue, that is,</p>
+                </div>
+            </div>
```

Comment:
this `</div>` closes the page--remove it. Same problem on line 1309.  more info at https://dig-eg-gaz.github.io/how-to/digitization/polished-tei-instructions/#1-get-the-page-hierarchy-right